python/mage/link_prediction/predictors/MLPPredictor.py

```diff
@@ -0,0 +1,65 @@
+from typing import Dict, Tuple
+import torch
+import torch.nn.functional as F
+import dgl
+from mage.link_prediction.constants import (
+    Predictors
+)
+
+class MLPPredictor(torch.nn.Module):
+    def __init__(self, h_feats: int) -> None:
+        super().__init__()
+
+        self.W1 = torch.nn.Linear(h_feats * 2, h_feats)
+        self.W2 = torch.nn.Linear(h_feats, 1)
```

Comment:
Did you add somewhere initialization of weights? Maybe this can help more with prediction results.

Reply:
Torch initializes them automatically in the following way:
 U(−k,k)\mathcal{U}(-\sqrt{k}, \sqrt{k})U(−k
​,k
​), where k=1in_featuresk = \frac{1}{\text{in\_features}}k=in_features1​